asminc/pet.inc

```diff
@@ -59,7 +58,25 @@ BRKVec          := $0092
 NMIVec          := $0094
 
 ; ---------------------------------------------------------------------------
-; I/O: 6522 VIA2
+; I/O: 6520 PIA1
```

Comment:
GitHub's diff is confused by two changes happening at the same location:
1.  I inserted the definition of PIA1 here.
2.  I corrected the comment to read "VIA" instead of "VIA2" (see line 79 below).